msbuild/Xamarin.iOS.Tasks.Windows/Xamarin.iOS.Common.After.targets

```diff
@@ -79,9 +79,6 @@ Copyright (C) 2011-2013 Xamarin. All rights reserved.
 			Sources="$(DeviceSpecificOutputPath)$(_AppBundleName)$(AppBundleExtension)"
 			OutputFile="$(DeviceSpecificOutputPath)AppBundle\$(_AppBundleName).zip"
 			WorkingDirectory="$(DeviceSpecificOutputPath)AppBundle" />
-		<!--Copy Zip from Mac-->
-		<Message Text="$(DeviceSpecificOutputPath)AppBundle\$(_AppBundleName).zip" />
-		<CopyFileFromBuildServer Condition="'$(MtouchTargetsEnabled)' And '$(IsMacEnabled)' == 'true'" SessionId="$(BuildSessionId)" File="$(DeviceSpecificOutputPath)AppBundle\$(_AppBundleName).zip" />
```

Comment:
Why is this deleted? It looks like that prevents the entire task from working as expected.

Reply:
The Zip task already takes care of copying the file back to Windows if it's executed remotely. See: https://github.com/dotnet/macios/blob/main/msbuild/Xamarin.MacDev.Tasks/Tasks/Zip.cs#L45C5-L47C67

That's why in the new target that I added, I use the Zip file and don't use the `CopyFileFromBuildServer`, and it works.

Am I missing something?

Reply:
Ah, I see, I didn't connect the dots!